collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -32,7 +32,7 @@
     public static final String KEY_MARK_AS_FINALIZED            = "mark_as_finalized";
 
     // server
-            static final String KEY_CHANGE_ADMIN_PASSWORD       = "admin_password";
+    static final String KEY_CHANGE_ADMIN_PASSWORD               = "admin_password";
```

Comment:
That's too bad! I guess it'd be hard to write a rule to line things up on the `static`.